File: Estrutura_Condicional/C++/Ex1049/main.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
    string A, B, C;
    cin >> A >> B >> C;

    if (A == "vertebrado")
    {
        if (B == "ave")
        {
            if (C == "carnivoro")
            {
                cout << "aguia" << endl;
            }
            else
            {
                cout << "pomba" << endl;
            }
        }
        else
        {
            if (C == "onivoro")
            {
                cout << "homem" << endl;
            }
            else
            {
                cout << "vaca" << endl;
            }
        }
    }
    else
    {
        if (B == "inseto")
        {
            if (C == "hematofago")
            {
                cout << "pulga" << endl;
            }
            else
            {
                cout << "lagarta" << endl;
            }
        }
        else
        {
            if (C == "hematofago")
            {
                cout << "sanguessuga" << endl;
            }
            else
            {
                cout << "minhoca" << endl;
            }
        }
    }
    return 0;
}

File: Estrutura_Condicional/C#/Ex1049/Main.cs
using System;

namespace Ex1049
{
    class Program
    {
        static void Main(string[] args)
        {
            string A, B, C;
            A = Console.ReadLine();
            B = Console.ReadLine();
            C = Console.ReadLine();

            if (A == "vertebrado")
            {
                if (B == "ave")
                {
                    if (C == "carnivoro")
                    {
                        Console.WriteLine("aguia");
                    }
                    else
                    {
                        Console.WriteLine("pomba");
                    }
                }
                else
                {
                    if (C == "onivoro")
                    {
                        Console.WriteLine("homem");
                    }
                    else
                    {
                        Console.WriteLine("vaca");
                    }
                }
            }
            else
            {
                if (B == "inseto")
                {
                    if (C == "hematofago")
                    {
                        Console.WriteLine("pulga");
                    }
                    else
                    {
                        Console.WriteLine("lagarta");
                    }
                }
                else
                {
                    if (C == "hematofago")
                    {
                        Console.WriteLine("sanguessuga");
                    }
                    else
                    {
                        Console.WriteLine("minhoca");
                    }
                }
            }
        }
    }
}